angr/procedures/libc/fgets.py

```diff
@@ -1,6 +1,6 @@
 import angr
-from angr.storage.file import SimFile
-from angr.sim_type import SimTypeFd, SimTypeChar, SimTypeArray, SimTypeLength
```

Comment:
These lines were removed to satisfy the linter. I presume they predate the linter and there isn't any magic that would cause them to do things.